addon/hint/sql-hint.js

```diff
@@ -26,9 +26,37 @@
     return CodeMirror.resolveMode(mode).keywords;
   }
 
+  function getText(item) {
+    if (typeof item == 'string') {
+      return item;
+    }
+    return item.text;
```

Comment:
Shouldn't this be item.displayText?
Let's suppose I have a table object {columns:{....}, text: '{{table1}}', displayText:'table1'}
Then when I type 'ta' it does not return me anything. Am I missing something?

Reply:
> Shouldn't this be item.displayText?

No, this is used to get the actual completion text, not the label.